website/src/content/docs/blog/biome-wins-prettier-challenge.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: Biome formatter wins the Prettier challenge
+template: splash
+url: /blog/biome-wins-prettier-challenge
+description: Biome formatter now 95% compatible with Prettier
+summary: Biome formatter now 95% compatible with Prettier
+authorName: Emanuele Stoppa
+authorAvatar: https://avatars.githubusercontent.com/u/602478?v=4
+authorUrl: https://twitter.com/ematipico
+coauthorName: Biome Core Team, Biome Maintainers
+coauthorAvatar: /img/circle-indent-logo.svg
+pubDate: 2023-11-27
+layout: ../../../layouts/Blog.astro
+coverImage: ../../src/assets/prettier-challenge.png
+coverImageAltText: The Prettier challenge banner, with the Biome logo over it
+socialImage: ../../src/assets/prettier-challenge.png
+---
+
+With the release of Biome **`v1.4.0`**, we claim the bounty of the [Prettier challenge](https://console.algora.io/challenges/prettier)!
+
+With `v1.4.0`, you'll get a better formatter experience, more formatting options, new VSCode features, new sponsors and more!
+
+You can upgrade Biome by running the following command:
+
+```bash
+npm install --save-dev --save-exact @biomejs/biome@1.4.0
+pnpm update --save-exact @biomejs/biome@1.4.0
+yarn upgrade --exact @biomejs/biome@1.4.0
+```
+
+##  Better formatter
+
+Biome formatter has now **over 95% in terms of compatibility** against [Prettier](https://prettier.io/)!
+
+Merit of challenge that was launched by [Christopher Chedeau](https://twitter.com/Vjeux/status/1722733472522142022), one of the Prettier's creators.
+
+The challenge attracted the attention of many people, and some of them decided to contribute to Biome to claim part of the bounty. I did see something amazing: contributors had an amazing coordination, they took ownership of the tasks and delivered the solution in a matter of hours.
+
+I believe the main factors that made this possible are three:
+1. Money. It's a fact, and it's completely fine if someone decides to contribute only for earning a small stipend out of it.
+2. Communication. We used [GitHub](https://github.com/biomejs/biome/issues/720) as only medium of coordination. We provided information, instructions and help on how to deliver.
+3. Infrastructure. Biome relies on a solid testing infrastructure, built by [previous](https://github.com/MichaReiser) Rome Tools [employees](https://github.com/ematipico) and [contributors](https://github.com/IWANABETHATGUY/). It's able to catch every idempotency bug, provide granular diffs and warn the user if the emitted output is the different from the one emitted by Prettier.
```

Comment:
`idempotency bug` can be confusing for newcomers. I tthink we should use `unstable fromatting bug` or `reformat bug`?